### PR TITLE
Added support for sound maps in the loader.

### PR DESCRIPTION
**Pull request by Snailprincess**

### CODE REVIEW

**Comment by mgh** [comment]
I've rebased this branch against master for the tabs->spaces switch and force-pushed

File: src/ui/resource/loader.js [hunk 1]
```diff
@@ -156,8 +165,15 @@ var Loader = Class(Emitter, function () {
         }
       }
 
+      var audioMap = this._audioMap;
+      var audioToLoad = {};
+      for (var uri in audioMap) {
+        if (uri.indexOf(pathPrefix) == 0) {
```
Comment (rogueSkib) [inline src/ui/resource/loader.js]: Style fix: always use triple equals; update the block above to use it as well - in general if you see something off-style or questionable in the engine, feel free to fix it.

File: src/ui/resource/loader.js [hunk 2]
```diff
@@ -101,6 +103,13 @@ var Loader = Class(Emitter, function () {
     }, this);
   };
 
+
+  this.addAudioMap = function (map) {
+    Object.keys(map).forEach(function(name) {
```
Comment (rogueSkib) [inline src/ui/resource/loader.js]: Same spacing fix after function here

**Comment by rogueSkib** [comment]
Looks good, thanks!

**Pull request merged**